Expand tranmission and asymptomatic spread (#635)
greenelab/covid19-review: content/08.transmission.md

## Transmission and Susceptibility

### Mechanism of Transmission

Person-to-person viral transmission of a virus can occur through several possible mechanisms.
The primary mechanisms associated with respiratory viruses are contact, droplet, and aerosol transmission [@doi:10.1097/QCO.0000000000000563].
Contact transmission can occur through either direct contact with a contagious person or indirect contact with active viral particles on a contaminated surface [@doi:10.4415/ANN_13_02_03].
This latter mode of transmission is also called fomite transmission [@url:https://www.who.int/news-room/commentaries/detail/transmission-of-sars-cov-2-implications-for-infection-prevention-precautions].
Viral particles can enter the body if they then come in contact with the oral, nasal, eye, or other mucus membranes [@doi:10.4415/ANN_13_02_03].

> Droplet transmission occurs when a contagious individual sneezes, coughs, or exhales and produces a respiratory droplet that can contain a large number of viral particles [@doi:10.4415/ANN_13_02_03].

Reviewer: ```suggestion
Droplet transmission occurs when a contagious individual sneezes, coughs, or exhales and produces respiratory droplets that can contain a large number of viral particles [@doi:10.4415/ANN_13_02_03].
```

> Contact with the droplets can occur either through direct exposure to the droplets, such as breathing in droplets produced by a sneeze, or exposure to particles that have settled on a surface [@doi:10.4415/ANN_13_02_03].

Reviewer: ```suggestion
Contact with these droplets can occur either through direct exposure to the droplets, such as breathing in droplets produced by a sneeze, or exposure to particles that have settled on a surface [@doi:10.4415/ANN_13_02_03].
```

Droplet and/or contact transmission are both well-accepted modes of transmission for many viruses associated with common human illness, including influenza and rhinovirus [@doi:10.4415/ANN_13_02_03]. 
The extent to which aerosol transmission contributes to the spread of respiratory viruses is less clear.
In influenza A, for example, viral particles can be detected in aerosols produced by infected individuals, but the extent to which these particles drive the spread of influenza A infection remains under debate [@doi:10.4415/ANN_13_02_03; @doi:10.1097/QCO.0000000000000563; @doi:10.3201/eid1301.061202; @doi:10.1038/s41598-019-38825-y; @doi:10.1093/femsre/fuv039].
Regardless of its role in the spread of influenza A, however, aerosol transmission likely played a role in outbreaks such as the 1918 Spanish Influenza (H1N1) and 2009 "swine flu" (pH1N1) [@doi:10.1093/femsre/fuv039].
Contact, droplet, and aerosol transmission are therefore all worth evaluating when considering possible modes of transmission for a respiratory virus like SARS-CoV-2.

#### Transmission of HCoV

> All three of these mechanisms have been identified as playing a role in the transmission of HCoV [@doi:10.4415/ANN_13_02_03], including the highly pathogenic coronaviruses SARS-CoV and MERS-CoV [@doi:10.1016/B978-0-12-385885-6.00009-2; @doi:10.1038/nrmicro.2016.81].

Reviewer: ```suggestion
All three of these mechanisms have been identified as contributors to the transmission of HCoV [@doi:10.4415/ANN_13_02_03], including the highly pathogenic coronaviruses SARS-CoV and MERS-CoV [@doi:10.1016/B978-0-12-385885-6.00009-2; @doi:10.1038/nrmicro.2016.81].
```

While droplet-based and contact transmission were initially considered to be the primary modes by which SARS-CoV-2 spread [@doi:10.1038/s41368-020-0075-9], as additional information emerged, the possibility of aerosol transmission was also raised [@doi:10.1126/science.abc6197; @doi:10.1056/NEJMc2004973; @doi:10.1093/cid/ciaa939].
For example, the detection of SARS-CoV-2 viral particles in air samples taken from hospitals treating COVID-19 patients led to the concern that the virus could also spread via aerosols [@doi:10.1038/s41586-020-2271-3].
The stability of the virus both in aerosols and on a variety of surfaces appeared similar to that of SARS-CoV [@doi:10.1056/NEJMc2004973], and fomite transmission may play a role in transmission (e.g., [@doi:10.3201/eid2606.200412]).

> However, while the possibility of aerosol transmission seems plausible, the evidence suggests that droplet and contact transmission are the dominant mechanisms driving the spread of the virus [@doi:10.1001/jama.2020.12458].

Reviewer: So the one thing I would say is that, while fomite transmission is possible, I am not aware of any specific incidences where it was proven. Hence why the emphasis of washing groceries and allowing take aways and food deliveries didn't become a major issue.

Author: I agree! Let me see if I can find some good sources on this and I'll tag you again. Thank you so much @RLordan!

As more estimates of the duration of viral shedding are released, they are beginning to converge around approximately three weeks from first positive PCR test and/or onset of symptoms (which, if present, are usually identified within three days of the initial PCR test).
For example, viral shedding was observed for up to 28 days following symptom onset in a second study [@doi:10.1038/s41586-020-2196-x] and for one to 24 days from first positive PCR test with a median of 12 days [@doi:10.1001/jama.2020.3204].
On the other hand, almost 70% of patients were reported to still have symptoms at the time that viral shedding ceased, although all symptoms reduced in prevalence between onset and cessation of viral shedding (CVS) [@10.1016/j.jcv.2020.104577].
They also reported that the median time that elapsed between the onset of symptoms and CVS was 23 days and between first positive PCR test and CVS was 17 days [@10.1016/j.jcv.2020.104577]. 

> This study reported that symptom onset predated the first positive PCR test by an average of three days, however, which suggests that there may be some methodological differences between it and related studies.

Reviewer: ```suggestion
This study reported that symptom onset predated the first positive PCR test by an average of three days, however, this suggests that there may be some methodological differences between it and related studies.
```

Author: ```suggestion
The fact that this study reported symptom onset to predate the first positive PCR test by an average of three days, however, suggests that there may be some methodological differences between it and related studies.
```

> An analysis of residents of a nursing home with a known SARS-CoV-2 case found similar rates of SARS-CoV-2 positive individuals who were asymptomatic and did not (25.5) or did (23.1) later develop symptoms, those who had atypical symptoms (24.2), and those who had typical symptoms (24.8) [@doi:10.1056/NEJMoa2008457].

Reviewer: The meaning of the brackets wasn't clear here, I went the the paper and it said it was the cycle thresholds. Is that required?

Author: I'm almost sure these are percentages... not sure what was wrong with me while writing this 😬  I'll double-check and add units!

Reviewer: HaHa no worries :)

Later analyses have sought to estimate the proportion of infections that could be traced back to a presymptomatic or asymptomatic individual.
Estimates of the proportion of individuals with asymptomatic infections have varied widely.
The proportion of asymptomatic individuals on board the Diamond Princess cruise ship, which was the site of an early COVID-19 outbreak, was estimated at 17.9% [@doi:10.2807/1560-7917], 
In contrast, a model using the prevalence of antibodies among residents of Wuhan, China estimated a much higher of asymptomatic cases, at approximately 7 in 8, or 87.5% [@doi:10.1093/pcmedi/pbaa032/5901532].
An estimate of the population of care homes in London found that, among the residents (median age 85), the rate of asymptomatic infection was 43.8%, and among the caretakers (median age 47), the rate was 49.1% [@doi:10.1016/j.eclinm.2020.100533]. 
The duration of viral shedding may also be longer in individuals with asymptomatic cases of COVID-19 compared to those who do show symptoms [@doi:10.1038/s41591-020-0965-6].
As a result, the potential for individuals who do not know they have COVID-19 to spread the virus is of significant concern.

> In the two cities studied to estimate incubation period, an estimated 40-50% and 60-80%, respectively, were estimated to be caused by contact with an asymptomatic individuals [@doi:10.7554/eLife.57149].

Reviewer: ```suggestion
In Singapore and Tianjin, two cities studied to estimate incubation period, an estimated 40-50% and 60-80% of cases respectively, were estimated to be caused by contact with an asymptomatic individuals [@doi:10.7554/eLife.57149].
```

Reviewer: I don't know if the comment before respectively is needed, correct me if I'm wrong :)

An analysis of viral spread in the Italian town of Vo', which was the site of an early COVID-19 outbreak, revealed that 42.5% of cases were asymptomatic and that the rate was similar across age groups [@doi:10.1038/s41586-020-2488-1].

> They argued that the town's lock down was imperative for controlling the spread of COVID-19 because it isolated the asymptomatic individuals.

Reviewer: ```suggestion
They argued that the town's lockdown was imperative for controlling the spread of COVID-19 because it isolated the asymptomatic individuals.
```

While more models are likely to emerge to better explore the effect of asymptomatic individuals on SARS-CoV-2 transmission, these results suggest that strategies for identifying and containing asymptomatic but contagious individuals are important for managing community spread. 

> <!-- Most credible IFR estimates I've seen are between 0.5-1.5%, depending on the population. CFRs will be greater and vary a lot.-->

Author: This comment was here and I'm not sure what I'm supposed to do with it -- suggestions welcome!

Reviewer: Infection fatality rate/ratio... 

Was it @dziakj1 mentioned this?

Reviewer: I guess we should mention the known IFR currently and maybe that initial estimates were high at 5% but that over time and with mitigation measures it reduced to somewhere in that range.

Author: Hmm this actually might be able to move down into the susceptibility edits that I'm expanding on right now... I'll try to work it in there!
